src/app/users/users-billing-info/users-billing-info.component.ts

```diff
@@ -10,13 +15,45 @@ import { SharedService } from '../../store/services';
 })
 export class UsersBillingInfoComponent implements OnDestroy, OnInit {
 
-  constructor(private sharedService: SharedService) { }
-
+  constructor(private sharedService: SharedService, private store: Store<AuthState>, private formBuilder: FormBuilder) { }
+  private cardNumber;
+  billingForm: FormGroup;
+  private expiryMonth = 'Month';
```

Comment:
Hi Kinji, you cannot use private variables inside template, this will cause the prod build to fail.
please mark those variables as public.